packages/runtime/src/widgets/Form/MultiSelect.tsx

```diff
@@ -127,12 +128,11 @@ const MultiSelect: React.FC<MultiSelectProps> = (props) => {
   };
 
   const handleSearch = (value: string): void => {
-    if (
-      values?.options.some((option) =>
-        option.label.toString().toLowerCase().startsWith(value.toLowerCase()),
-      )
-    )
-      setNewOption("");
+    const isOptionExist = values?.options.some((option) =>
+      option.label.toString().toLowerCase().startsWith(value.toLowerCase()),
+    );
+
+    if (isOptionExist || values?.mode !== "tags") setNewOption("");
```

Comment:
Why is this required if the antd mode already handles the new option?

Reply:
To remove this bottom message when we disable the tags mode, we need this condition

<img width="1170" alt="image" src="https://github.com/EnsembleUI/ensemble-react/assets/31651612/542b0c35-d85e-4665-8028-d8ac9548208e">